src/auth/jwtaccess.ts

```diff
@@ -24,6 +25,9 @@ export class JWTAccess {
   key?: string|null;
   projectId?: string;
 
+  private cache =
+      LRU<string, RequestMetadataResponse>({max: 500, maxAge: 60 * 60 * 1000});
```

Comment:
question: why is 500 a reasonable number here?

Reply:
I will admit to choosing this at complete random.  Totally open to a better number :)